Firmware/messages.cpp

```diff
@@ -187,7 +187,7 @@ extern const char MSG_LOAD_ALL[] PROGMEM_I1 = ISTR("Load All"); ////MSG_LOAD_ALL
 extern const char MSG_NOZZLE_CNG_MENU [] PROGMEM_I1 = ISTR("Nozzle change");////MSG_NOZZLE_CNG_MENU c=18
 extern const char MSG_NOZZLE_CNG_READ_HELP [] PROGMEM_I1 = ISTR("For a Nozzle change please read\nprusa.io/nozzle-mk3s");////MSG_NOZZLE_CNG_READ_HELP c=20 r=4
 extern const char MSG_NOZZLE_CNG_CHANGED [] PROGMEM_I1 = ISTR("Hotend at 280C! Nozzle changed and tightened to specs?");////MSG_NOZZLE_CNG_CHANGED c=20 r=6
-
+extern const char MSG_REPRINT [] PROGMEM_I1 = ISTR("Reprint"); ////MSG_REPRINT c=7
```

Comment:
As this is a submenu the allowed char length is `c=18`

Reply:
Ok, I suppose that I had to specify the longer length, I will do the change.

Reply:
The pot file defines all messages and po files are based on it.
That is why we have the two update-po/t.py scripts to keep them correct.
Editing the po/t files manually can break things badly.

Reply:
Reminder, that `c=7` is still present, should be `c=18`